packages/amazonq/README.md

```diff
@@ -1,57 +1,62 @@
 ## Inline code suggestions
 
+[![Marketplace Version](https://img.shields.io/vscode-marketplace/v/AmazonWebServices.amazon-q-vscode.svg)
+![Marketplace Downloads](https://img.shields.io/vscode-marketplace/d/AmazonWebServices.amazon-q-vscode.svg)](https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.amazon-q-vscode)
+
 Code faster with inline code suggestions as you type.
 
-![Inline code suggestion demo](https://github.com/aws/aws-toolkit-vscode/raw/HEAD/docs/marketplace/vscode/amazonq/inline.gif)
+## Getting Started
+
+**Free Tier** - create or log in with an AWS Builder ID (no AWS account needed!).
+
+**Pro Tier** - if your organization is on the Amazon Q Developer Pro tier, log in with single sign-on.
+
+![Authentication gif](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/master/docs/marketplace/vscode/amazonq/auth-Q.gif)
+
+## Features
+
+![Inline code suggestion demo](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/master/docs/marketplace/vscode/amazonq/inline.gif)
 
 <sub>[_15+ languages supported including Python, TypeScript, Rust, Terraform, AWS Cloudformation, and more_](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/q-language-ide-support.html)</sub>
 
-## Chat
+### Chat
 
 Generate code, explain code, and get answers to questions about software development.
 
-![Generate code using chat](https://github.com/aws/aws-toolkit-vscode/raw/HEAD/docs/marketplace/vscode/amazonq/chat.gif)
+![Generate code using chat](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/master/docs/marketplace/vscode/amazonq/chat.gif)
 
-## Security scans
+### Security scans
 
 Analyze and fix security vulnerabilities in your project.
 
-![Fix security vulnerability demo](https://github.com/aws/aws-toolkit-vscode/raw/HEAD/docs/marketplace/vscode/amazonq/security-scan.gif)
+![Fix security vulnerability demo](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/master/docs/marketplace/vscode/amazonq/security-scan.gif)
 
 <sub>[_10 languages supported including Python, TypeScript, C#, and more_](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/security-scans.html)</sub>
```

Comment:
why is this forced to be so small? it doesn't look great

![image](https://github.com/aws/aws-toolkit-vscode/assets/55561878/49dc0acf-5a0c-419b-982c-31c611aa0317)